metadata.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+import os
+
+# Path of .csv file
+input_path = input("Enter path of .csv metadata file: ")
+output_dir = input("Enter directory for output (leave blank for current directory): ")
+
+# Exit if path doesn't exist
+if not os.path.exists(input_path):
+    print("Invalid input path, exiting...")
+    exit()
```

Comment:
`pd.read_csv` on line 24 should normally fail with `FileNotFoundError: [Errno 2] No such file or directory:`, which is a better place to raise the error. Also, see the comment on line 25.